solution/ui/regulations/js/src/components/RelatedRuleList.vue

```diff
@@ -169,6 +169,12 @@ export default {
             return rule.title || rule.description;
         },
         type(rule) {
+            if (rule.withdrawal) {
+              return "WD"
```

Comment:
Our existing JS indent style is four spaces.  You have two-space indents here and below